_dashboards/admin-ui-index/index-management.md

```diff
@@ -80,43 +86,43 @@ To apply a policy, select the index to which you want to apply the policy in the
 
 <img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/apply-policy.PNG" alt="User interface showing apply policy prompt">
 
-#### Close
+### Close
```

Comment:
Should this be "Close index" to match the name of the operation? Globally ensure all headings are in the correct format and/or match the name of the operation.